README.md

```diff
@@ -74,3 +74,10 @@ process.exit(result ? 0 : 1);
   a number before evaluation, which makes it easier for callers to pass
   environment variables as-is. NaN values will silently be assigned the default
   value.
+* updateExpectedFailureFiles?: boolean Used in local development to
+  automatically revise expected-failure files after making code changes that
+  fix test failures, removing tests that were expected to fail but now pass from
+  the files. This option does not add newly failing tests to the expected-failure
+  files - this must be done manually. Do not use this option in CI, instead we
+  recommend this option be hooked up to an optional CLI flag for ease of use,
+  but we do not recommend using this option in CI.
```

Comment:
Using this in CI won't do anything, because the files will be thrown away. (And might not even be writable in some CI envts?)  So maybe just remove this line and end the previous line with a period?

Reply:
Revised - PTAL.